src/main/scala/RigPlugin.scala

```diff
@@ -237,31 +226,22 @@ object common {
     val thisRef = ex.get(thisProjectRef)
     val slug = (travisRepoSlug in thisRef).get(ex.structure.data).flatten
     val jobNumber = (travisJobNumber in thisRef).get(ex.structure.data).flatten
-    val state1 = Command.process(s"sonatypeOpen ${slug.getOrElse("unknown")}-${jobNumber.getOrElse("0.0")}", state)
-
-    val ex1 = Project.extract(state1)
-    val thisRef1 = ex1.get(thisProjectRef)
-    reapply(ex1.structure.allProjectRefs.flatMap { proj => Seq(
-      publishTo in proj := (publishTo in thisRef1).get(ex1.structure.data).get,
-      sonatypeStagingRepositoryProfile in proj := (sonatypeStagingRepositoryProfile in thisRef).get(ex1.structure.data).get
-    )}, state)
+    releaseStepCommand(s"sonatypeOpen ${slug.getOrElse("unknown")}-${jobNumber.getOrElse("0.0")}")(state)
   }
 
-  val releaseAndClose: ReleaseStep = ReleaseStep(action = state1 => {
-    val ex1 = Project.extract(state1)
+  val releaseAndClose: ReleaseStep = ReleaseStep(action = state => {
+    val ex1 = Project.extract(state)
     val thisRef = ex1.get(thisProjectRef)
     val repoId = (sonatypeStagingRepositoryProfile in thisRef).get(ex1.structure.data).get.repositoryId
-
-    Command.process(s"sonatypeRelease $repoId", state1)
+    state.log.info(s"Closing staging repo ${repoId}")
+    releaseStepCommand(s"sonatypeDrop $repoId")(state)
```

Comment:
🤦‍♂️  This should be `sonatypeRelease`.  This is from my local testing so I didn't pollute Central.